dis_snek/models/discord_objects/invite.py

```diff
@@ -81,4 +84,6 @@ def _process_dict(cls, data: Dict[str, Any], client: "Snake") -> Dict[str, Any]:
 
     @property
     def link(self) -> str:
+        if self.scheduled_event:
+            return f"https://discord.gg/{self.code}?event={self.scheduled_event}"
         return f"https://discord.gg/{self.code}"
```

Comment:
```py
        return (
            f"https://discord.gg/{self.code}" if self.scheduled_event
            else f"https://discord.gg/{self.code}"
        )
```

This isn't really *required* per-se, but it would be more consistent with how black generalizes formatting for ternary returns.

Reply:
I personally feel like the current way is more readable.  No need to turn it into a ternary.

Reply:
Ya I am going to agree with @silasary. The current method is just better to look at.